prow/config.yaml

```diff
@@ -1002,6 +1002,71 @@ presubmits:
         - "--upload=gs://kubernetes-jenkins/pr-logs"
 
   kubernetes/kubernetes:
+  - name: ci-kubernetes-dind-e2e
```

Comment:
assume this is a periodic job? then move it to periodics, rather than put this in presubmit :-)

Reply:
I intend it to eventually be a blocking presubmit. What made you think it's periodic?

Reply:
then the job name should be `presubmit-` instead of `ci-` IMO